src/packages/shared-types/action-types/new-submission.ts

```diff
@@ -11,6 +11,8 @@ export const onemacSchema = z.object({
   submitterEmail: z.string(),
   attachments: z.array(attachmentSchema).nullish(),
   raiWithdrawEnabled: z.boolean().default(false),
+  proposedEffectiveDate: z.number().nullish(),
```

Comment:
Ok, so I did a bunch of thinking about this... what I arrived at is: 
- the initial-submission emails are a record of what the state submitter submitted.... to that end, we should send out what the state submitter chose, NOT what got entered into SEA Tool.
- The CPOC and SRT requirements are different, because they are only needed for packages that already exist, so the CPOC and SRT will already be in the OpenSearch data and it does not matter (as much) if they get changed in between, since, at the time of the submission, those are the valid values for CPOC and SRT (or whatever else we pull from the package).

That being said, I made these choices to complete the story as written without regard to what changes will need to be made based on the architecture adaptations.  I figured... get something out there and we can modify it later :)